gBrogue Source/PlatformCode/curses-platform.c

```diff
@@ -74,7 +74,7 @@ static void curses_plotChar(uchar ch,
 	case LOZENGE_CHAR: ch = 'o'; break;
 	case CROSS_PRODUCT_CHAR: ch = 'x'; break;
 
-	case STATUE_CHAR: ch = '5'; break;
+	  //	case STATUE_CHAR: ch = '5'; break;
 //same as MOLOCH_CHAR
```

Comment:
STATUE_CHAR and MOLOCH_CHAR are the same in gBrogue so this is just a fix to keep the curses target compiling